qCC/db_tree/ccPropertiesTreeDelegate.cpp

```diff
@@ -60,6 +60,7 @@
 #include <QCheckBox>
 #include <QComboBox>
 #include <QHBoxLayout>
+#include <QLineEdit>
```

Comment:
Why was this needed for the inheritance change?

Reply:
Ah :-)

Because these UI header files include half of Qt (which is one of the reasons to move to the member instead of inheritance), QLineEdit was included indirectly in this file.

So when the UI header is moved out of the .h, there may be some fixes like this.